### PR TITLE
Snowflake bulk load

### PR DESCRIPTION
**Pull request by eeshansim**

### CODE REVIEW

File: connectors/snowflake/bulk-load-s3.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+---
+title: Workato connectors - Snowflake Bulk Load
+date: 2019-01-10 06:10:00 Z
+---
+
+# Snowflake - Bulk Load from external stage action
+
+Load a file from an external stage Amazon S3 bucket as an external source into a target table. This action uses the [COPY](https://docs.snowflake.net/manuals/sql-reference/sql/copy-into-table.html#output) command to load data directly from an external source to a target table.
+
+This action will execute the load and wait for completion before moving onto the next step. Load time depends on the size of source file, number of columns, additional validation in the target table and network speed (faster if loading data from S3 to an AWS-deployed Snowflake instance). **1 GB** CSV file with **30 columns** and **3 million rows** will take 60 seconds.
```
Comment (graceliching) [inline connectors/snowflake/bulk-load-s3.md]: Is this time of 60s an estimate or a fact?

File: connectors/snowflake/bulk-load-s3.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+      <td>Errors seen</td>
+      <td>Number of rows with error in the source file.</td>
+    </tr>
+    <tr>
+      <td>First error</td>
+      <td>Error details of the first error in the source file.</td>
+    </tr>
+    <tr>
+      <td>First error line</td>
+      <td>Line number of the first row that caused an error.</td>
```
Comment (graceliching) [inline connectors/snowflake/bulk-load-s3.md]: For this, should there be an additional paragraph of how to use this pill as a way to restart the same job from this line number onwards?

File: connectors/snowflake/bulk-load-s3.md (hunk 1 above)
Comment (eeshansim) [reply connectors/snowflake/bulk-load-s3.md]: Fact, tested it manually.

File: connectors/snowflake/bulk-load-s3.md (hunk 2 above)
Comment (eeshansim) [reply connectors/snowflake/bulk-load-s3.md]: There is no proper way to do that, this is mainly for reporting. Typical CSV files will be too big to parse and pick out specific subset of the content. It is simpler to just re-run the entire job.

**Pull request merged**